### DISCUSSION TITLE
HA not finding integration myicomfort

### DISCUSSION THREAD

**Discussion by jldesign** [thevoltagesource/LennoxiComfort#35] [Q&A]
I am running HA container v2021.11.5. I have followed your guide to adding myicomfort into /config/custom_components/ with the minimal settings. when I restart HA, I get the following error.

` The system cannot restart because the configuration is not valid: Platform error climate.myicomfort - Integration 'myicomfort' not found. `

Any advise/help would be greatly appreciated.

**Comment by thevoltagesource** [comment]
Due to changes in the way HA loads components and validates configs, I need to update the instructions.  To get this up and running, remove the configuration entries from your configuration.yaml file and restart Home Assistant.  Then, put the config lines back in the configuration.yaml file and restart one more time. 

That reboot in the middle allows HA to load the custom component and be aware of the available integration so that the config validation will pass.

**Reply by jldesign** [reply]
I # out the entries in the config, restarted, un-#, restarted and still the same results. :(

**Reply by thevoltagesource** [reply]
When you restart home assistant, do you see a warning in the logs regarding the inclusion of the custom component?  After a restart, if I go to in the UI to Configuration -> Logs and click the Load Full Home Assistant Log button, this is one of the top three entries.

```2021-11-29 14:33:07 WARNING (SyncWorker_2) [homeassistant.loader] We found a custom integration myicomfort which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant```

If you are not seeing this then you need to verify that all the files were copied to the correct location.  File layout should look like this:

```
jacob@raspberrypi:/mnt/ha $ ls custom_components/myicomfort/
climate.py  __init__.py  manifest.json  __pycache__
```

**Reply by thevoltagesource** [reply]
Were you able to get this resolved?  You might consider installing via HACS.
